src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -13618,21 +13632,41 @@ void Player_Action_8084FBF4(Player* this, PlayState* play) {
     func_8002F8F0(&this->actor, NA_SE_VO_LI_TAKEN_AWAY - SFX_FLAG + this->ageProperties->unk_92);
 }
 
-s32 func_8084FCAC(Player* this, PlayState* play) {
+#if OOT_DEBUG
+/**
+ * Updates the "Noclip" debug feature, which allows the player to
+ * fly around anywhere in the world and clip through any collision.
+ * It can be enabled with two seperate button combos:
+ * A + L + R held down and then B
+ * or
+ * L held down and then D-pad right
+ *
+ * To control Noclip mode:
+ * - Move horizontally with the 4 D-pad directions
+ * - Move up with B
+ * - Move down with A
+ * - Hold R to move faster
+ *
+ * With Noclip enabled, another button combination can be pressed to set all "temp clear" flags
+ * in the current room. To do so Hold L and press D-pad right.
+ *
+ * @return  true if Noclip is disabled, false if enabled
```

Comment:
Maybe would be clearer to say it returns whether Link should update normally (since that's how it's used)

Reply:
It would make more sense that way yeah. But in a vacuum the function doesnt know what it indicates with regards to player's update, just the state of noclip. So it feels weird to be talking about the implication on players update in this function.
Would like to hear others thoughts as well

Reply:
as much as I hate the double negative I agree it's better to reflect what the function does than what it's used for